lib/ae_mdw_web/controllers/name_controller.ex

```diff
@@ -97,7 +95,8 @@ defmodule AeMdwWeb.NameController do
   def owned_by(conn, %{"id" => owner} = params),
     do:
       handle_input(conn, fn ->
-        owned_by_reply(conn, Validate.id!(owner, [:account_pubkey]), expand?(params))
+        active? = params |> Map.get("active", "true") |> String.to_existing_atom()
```

Comment:
This will return a 500 for any value that does not exist as an `atom`. We should probably use `Map.get("active", "true") == "true"` instead

Reply:
done